core/storage/suggestion/gae_models.py

```diff
@@ -1990,3 +2022,1459 @@ def export_data(
+                SortChoices.SORT_KEY_INCREASING_ACCURACY.value
+            ]:
+                cls.overall_accuracy,
+            constants.CD_ADMIN_STATS_SORT_OPTIONS[
+                SortChoices.SORT_KEY_DECREASING_SUBMISSIONS.value
+            ]:
+                -cls.submitted_questions_count,
+            constants.CD_ADMIN_STATS_SORT_OPTIONS[
+                SortChoices.SORT_KEY_INCREASING_SUBMISSIONS.value
+            ]:
+                cls.submitted_questions_count
+        }
+
+        sort = -cls.recent_performance
+        if sort_by is not None:
+            sort = sort_options_dict[sort_by]
+
+        # The first sort property must be the same as the property to which
+        # an inequality filter is applied. Thus, the inequality filter on
+        # last_activity can not be used here and we have implemented it below.
+        # Learn more about this here
+        # https://cloud.google.com/appengine/docs/legacy/standard/go111/datastore/query-restrictions#properties_used_in_inequality_filters_must_be_sorted_first.
+        if topic_ids is not None:
+            sort_query = cls.query(
+                datastore_services.all_of(
+                    cls.topic_ids_with_question_submissions.IN(topic_ids)
+                )).order(sort)
+        else:
+            sort_query = cls.get_all().order(sort)
+
+        sorted_results: List[
+            QuestionSubmitterTotalContributionStatsModel] = []
+        today = datetime.date.today()
+
+        if num_days_since_last_activity is not None:
+            last_date = today - datetime.timedelta(
+                days=num_days_since_last_activity)
+            next_offset = offset
+            while len(sorted_results) < page_size:
+                result_models: Sequence[
+                    QuestionSubmitterTotalContributionStatsModel] = (
+                    sort_query.fetch(
+                        NUM_MODELS_PER_FETCH, offset=next_offset))
+                if not result_models:
+                    break
+                for result_model in result_models:
+                    next_offset += 1
+                    if result_model.last_contribution_date >= last_date:
+                        sorted_results.append(result_model)
+                        if len(sorted_results) == page_size:
+                            break
+        else:
+            sorted_results = list(sort_query.fetch(page_size, offset=offset))
+            next_offset = offset + len(sorted_results)
+
+        # Check whether we have more results.
+        next_result_model: Sequence[
+            QuestionSubmitterTotalContributionStatsModel] = (
+                sort_query.fetch(offset=next_offset))
+        more: bool = len(next_result_model) != 0
+
+        return (
+            sorted_results,
+            next_offset,
+            more
+        )
+
+    @classmethod
+    def get_deletion_policy(cls) -> base_models.DELETION_POLICY:
+        """Model contains corresponding to a user: contributor_id."""
```

Comment:
This sentence doesn't scan, and I commented on this before.

When addressing a comment, please check other cases which might need to be fixed similarly; don't expect the reviewer to point all those cases out for you. (They often will, to an extent, but you should be doing the final audit.)

Reply:
Changed this as well as other docstrings in this file to be accurate, Thanks.